solidity/dashboard/src/stories/WalletConnectModal.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import centered from "@storybook/addon-centered/react"
+import { WalletConnectConnector } from "../connectors"
+import WalletConnectModal from "../components/WalletConnectModal"
+
+export default {
+  title: "WalletConnectModal",
+  component: WalletConnectModal,
```

Comment:
Each story for modal has transparent background. Can we set white background by default?